### PR TITLE
SDPA-502 If a subservice is required then the parent service should automatically be set

### PR DESCRIPTION
**Pull request by mugurrus**
Sets the anpa category corresponding to the required subservice:
if a subservice field (defined in vocabularies) was declared as
required in a content profilebthen make sure that the corresponding
anpa category was added to the anpa_category field in the newly
created item. Without this value the anpa category field is not
displayed.

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

**Comment by gbbr** [comment]
The logic seems a bit complex, so a test would be nice if you have the time :-)

**Comment by gbbr** [comment]
Thanks for adding the test. Looks good 👍

File: scripts/superdesk-authoring/authoring.spec.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+fdescribe('authoring', function() {
```
Comment (petrjasek) [inline scripts/superdesk-authoring/authoring.spec.js]: seems like forgotten `fdescribe`

**Pull request merged**